cases/era5_2021.yaml

```
@@ -1,5 +1,6 @@
 # Settings needed for preprocessing
-input_folder: /data/volume_2/era5_2021
+#input_folder: /data/volume_2/era5_2021
+input_folder: /data/volume_2/ERA5
```

```suggestion
input_folder: /data/volume_2/ERA5
```
I'd suggest to just remove the outdated line. If we ever need it back, it is still available in the git history